go/pkg/utils/languages_file_handler.go

```diff
@@ -86,7 +86,7 @@ func getLanguagesProperties() schema.LanguagesProperties {
 func getLanguagesFilePath() (string, error) {
 	_, b, _, _ := runtime.Caller(0)
 	basepath := filepath.Dir(b)
-	langFilePath := filepath.Join(basepath, "..", "..", "languages.yml")
+	langFilePath := filepath.Join(basepath, "..", "..", "..", "resources", "languages.yml")
```

Comment:
I think it will require user to have a copy of the file when using the library. We need to find a way to embed the file in the Go binary

Reply:
I was trying to find some info about it because i also have a doubt but it worked fine before without doing any copy so maybe i don't need to do anything here as well. Maybe golang copy it automatically when imported in some other project. @kadel do you know how it works?

Reply:
We can create a Makefile and add scripts there and add some documentation in the README. By looking around it seems we cannot customize the go build command. But if we add the copy script + go build within a Makefile we could use that one which execute all steps in one shot.